crates/sui-graphql-rpc/tests/dot_move_e2e.rs

```diff
@@ -0,0 +1,441 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(feature = "pg_integration")]
+mod tests {
+    use std::{path::PathBuf, time::Duration};
+
+    use sui_graphql_rpc::{
+        config::{ConnectionConfig, ServiceConfig},
+        test_infra::cluster::{start_graphql_server_with_fn_rpc, wait_for_graphql_server},
+    };
+    use sui_graphql_rpc_client::simple_client::SimpleClient;
+    use sui_json_rpc_types::{ObjectChange, SuiTransactionBlockResponse};
+    use sui_move_build::BuildConfig;
+    use sui_types::{
+        base_types::{ObjectID, SequenceNumber},
+        digests::ObjectDigest,
+        move_package::UpgradePolicy,
+        object::Owner,
+        programmable_transaction_builder::ProgrammableTransactionBuilder,
+        transaction::{CallArg, ObjectArg, TransactionData},
+        Identifier, SUI_FRAMEWORK_PACKAGE_ID,
+    };
+    use tokio::time::sleep;
+
+    const DOT_MOVE_PKG: &str = "tests/dot_move/dot_move/";
+    const DEMO_PKG: &str = "tests/dot_move/demo/";
+    const DEMO_PKG_V2: &str = "tests/dot_move/demo_v2/";
+    const DEMO_PKG_V3: &str = "tests/dot_move/demo_v3/";
+
+    #[derive(Clone, Debug)]
+    struct UpgradeCap(ObjectID, SequenceNumber, ObjectDigest);
+
+    #[tokio::test]
+    async fn test_dot_move_e2e() {
+        let network_cluster = sui_graphql_rpc::test_infra::cluster::start_network_cluster(
+            ConnectionConfig::ci_integration_test_cfg(),
+            None,
+        )
+        .await;
+
+        let external_network_chain_id = network_cluster
+            .validator_fullnode_handle
+            .fullnode_handle
+            .sui_client
+            .read_api()
+            .get_chain_identifier()
+            .await
+            .unwrap();
+
+        println!("External chain id: {:?}", external_network_chain_id);
```

Comment:
Was it intentional to keep this here? If so, let's make it print to stderr, using `eprintln!` instead.

Reply:
The main reason is to track progress through logs (and get a feeling of where it is if a fail occurs)